src/pages/settings/Security/TwoFactorAuth/CodesPage.js

```diff
@@ -123,15 +123,15 @@ function CodesPage(props) {
                             )}
                         </View>
                     </Section>
-                    <FixedFooter style={[styles.twoFactorAuthFooter]}>
-                        <Button
-                            success
-                            text={props.translate('common.next')}
-                            onPress={() => Navigation.navigate(ROUTES.SETTINGS_2FA_VERIFY)}
-                            isDisabled={isNextButtonDisabled}
-                        />
-                    </FixedFooter>
                 </ScrollView>
+                <FixedFooter style={[styles.mtAuto, styles.pt2]}>
+                    <Button
```

Comment:
We already have defined the same `styles.twoFactorAuthFooter` so we should update that.

Reply:
Good catch! I replaced the other usage of `twoFactorAuthFooter` with the same as here to keep it standard, and then deleted it